src/setUnion.js

```diff
@@ -0,0 +1,25 @@
+export default function setUnion(arr1,arr2){
+
+let results = []
+const normalizeLength = Math.max(arr1.length, arr2.length)
+
+    for (let i = 0; i < normalizeLength; i++ ){
```

Comment:
Unindent these `for` loops to make it clear that they're not part of an inner function.